micronota/bfillings/tests/test_hmmer.py

```diff
@@ -0,0 +1,99 @@
+from tempfile import mkstemp
+from os import getcwd, remove, close
+from unittest import TestCase, main
+from functools import partial
+from skbio.util import get_data_path
+from burrito.util import ApplicationError
+from os.path import join
+from micronota.bfillings.hmmer import (HMMScan, hmmscan_fasta,
+                                       hmmpress_hmm)
```

Comment:
the import should be ordered by builtin module, dependency module, and module of this pkg (separated by a empty line between the three).